src/pages/WishList/WishList.js

```diff
@@ -1,7 +1,121 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
+import styled from "styled-components";
+import {
+  DetailWrapper,
+  DetailContain,
+  LeftMenu,
+  LeftMenuWrap,
+  DetailWrap,
+  DetailCont,
+} from "components/Detail/Detail";
+import { PageBack, BackBtn, DetailListWrap } from "components/Detail/LeftMenu";
+import DetailLeftItem from "./components/DetailLeftItem";
+import DetailRightItem from "./DetailRightItem";
+import {
+  PopWrap,
+  PopCont,
+  PopBtnBox,
+  BtnCancel,
+  BtnConfirm,
+} from "./components/Popup";
 
 const WishList = () => {
-  return <div>WishList</div>;
+  const [detail, setDetail] = useState([]);
+  const [count, setCount] = useState(1);
+  const [detailForRight, setDetailForRight] = useState();
+  const [popup, setPopup] = useState(false);
+
+  const handleLikeClick = () => {
+    setPopup(!popup);
```

Comment:
```suggestion
    setPopup((prev) => !prev);
```
boolean 값에 대해서 변동을 할 때, 위와 같이 함수형 업데이트로 구현해주세요~